### PR TITLE
Fix console warnings and leave the OLO flow the same 

### PR DESCRIPTION
**Pull request by svenjens**
Description 
This PR solves a console warning about nested elements. And readded some checks to make sure the OLO flow stays unchanged

Please make sure:
- [x] to add a label
- [x] to add one or more reviewers
- [x] you followed our checklist for [functional testing](https://github.com/Amsterdam/vergunningcheck/blob/master/TESTING.md)
- [x] you added the necessary documentation (either in the markdown files or inline)
- [x] you added the necessary automated tests

### CODE REVIEW

File: apps/client/src/components/RegisterLookupSummary.js [hunk 1]
```diff
@@ -43,31 +43,36 @@ const RegisterLookupSummary = ({
...
-        gutterBottom={hasSTTR ? 0 : 16}
-      >
-        {cityScape &&
-          `Het gebouw ligt in een beschermd stads- of dorpsgezicht.`}
-      </Paragraph>
+      {compact ? (
+        <AddressLine {...address} />
+      ) : (
+        <>
+          <TextToEdit>
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.js]: TextToEdit is a span (-; and should not be user here.

File: apps/client/src/components/RegisterLookupSummary.js [hunk 2]
```diff
@@ -43,31 +43,36 @@ const RegisterLookupSummary = ({
...
+          <ChangeAddressModal {...{ hasSTTR, setActiveState, slug }} />
+        </>
+      )}
+      {(monument || !hasSTTR) && (
+        <Paragraph data-testid={LOCATION_RESTRICTION_MONUMENT} gutterBottom={0}>
+          {monument
+            ? `Het gebouw is een ${monument.toLowerCase()}.`
+            : "Het gebouw is geen monument."}
+        </Paragraph>
+      )}
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.js:63]: Clever solution!

**Review by robinpiets — CHANGES_REQUESTED** [review]
Please improve

File: apps/client/src/components/RegisterLookupSummary.js [hunk 3]
```diff
@@ -43,31 +43,34 @@ const RegisterLookupSummary = ({
...
     <ComponentWrapper marginBottom={sttrFile ? "0" : null}>
-      <Paragraph gutterBottom={16}>
-        {compact ? (
+      {compact ? (
+        <AddressLine {...address} />
+      ) : (
+        <>
+          <Paragraph strong gutterBottom={0}>
+            Ingevoerd adres:
+          </Paragraph>
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.js:52]: This looks better and easier to understand:
```
{!compact && <Paragraph strong gutterBottom={0}>Ingevoerd adres:</Paragraph>}

<AddressLine {...address} />

{!compact && <ChangeAddressModal {...{ hasSTTR, setActiveState, slug }} />}
```

File: apps/client/src/components/RegisterLookupSummary.test.js [hunk 4]
```diff
@@ -30,7 +31,7 @@ describe("RegisterLookupSummary", () => {
     );
   };
 
-  it("renders correctly in STTR Flow", () => {
+  it("renders correctly in STTR Flow if monument", () => {
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.test.js:34]: 👍

File: apps/client/src/components/RegisterLookupSummary.test.js [hunk 5]
```diff
@@ -50,13 +51,70 @@ describe("RegisterLookupSummary", () => {
     });
   });
 
-  it("renders correctly in OLO Flow", () => {
+  it("renders correctly in STTR Flow if NOT a monument", () => {
```
Comment (robinpiets) [inline apps/client/src/components/RegisterLookupSummary.test.js:54]: 👍

**Review by robinpiets — APPROVED** [review]
Please fix the `<AddressLine />` rendering, see comment, and then you can merge (-:

File: apps/client/src/components/RegisterLookupSummary.js (hunk 3 above)
Comment (svenjens) [reply apps/client/src/components/RegisterLookupSummary.js:52]: This is fixed in an other PR

**Pull request merged**